packages/pnpm/test/monorepo/index.ts

```diff
@@ -226,6 +226,64 @@ test('Regression: topological order of packages with self-dependencies in monore
 
 })
 
+test('Regression: Do not get confused by filtered dependencies when searching for dependents in monorepo', async (t: tape.Test) => {
```

Comment:
no need to put `Regression` at the beginning.
you may put `recursive:`, I guess

Reply:
Okay.  I will remove 'Regression:'